src/3box.js

```diff
@@ -89,19 +89,14 @@ class Box extends BoxApi {
 
   async _load (opts = {}) {
     const address = await this._3id.getAddress()
-    const { rootStoreAddress, did } = address ? await this._getLinkedData(address) : {}
-    if (rootStoreAddress) {
-      await this.replicator.start(rootStoreAddress, did, { profile: true })
-      await this.replicator.rootstoreSyncDone
-      const authData = await this.replicator.getAuthData()
-      await this._3id.authenticate(opts.spaces, { authData })
-    } else {
-      await this._3id.authenticate(opts.spaces)
-      const rootstoreName = this._3id.muportFingerprint + '.root'
-      const key = (await this._3id.getPublicKeys(null, true)).signingKey
-      await this.replicator.new(rootstoreName, key, this._3id.DID)
-      this._publishRootStore(this.replicator.rootstore.address.toString())
-    }
+    const did = await this.accountLinks.read(address)
```

Comment:
What if no DID has  been linked to from this address yet? Seems like the flow would break then?